src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -171,29 +166,23 @@ export function DiscoverLegacy({
           showSearchBar={true}
           useDefaultBehaviors={true}
         />
-        <main className="container-fluid">
```

Comment:
We lost this `<main>` on the new page. Is there a different one being provided somewhere? If not, we need to bring this back.

Assuming we need to bring the `<main>` back, we should also move the `<h1>` that's above inside the `<main>` and label it the same way we do with the `<section>` tags below.

```tsx
<main aria-describedby={uniqueId}>
  <h1 id={uniqueId}>{savedSearch.title}</h1>
  {/* the rest of the page content */}
</main>
```

Reply:
Thx & done!